### PR TITLE
 Improved Button Responsiveness and Layout Adjustments

### LINKED ISSUE #9156: fix the responsive ness
Image
Image
in the above pic the header is not responsive
Image
in the above pic the red button and green outlined button are collapsed 
and the three buttons are not good at down the 3rd button should be the center.
Like this there are many issues i want to work on this issues .
assign this issue to me i will fix.

### PR DESCRIPTION
**Pull request by gurramkarthiknetha**
## Proposed Changes

- Fixes #9156 
image
here i changed the hap between the red button and green oulined button
- Change 2
-image
here i changed the responsive-ness wrap
- Change 3
- image
- here i improved the responsive ness of the buttons by align to center
- Change 4
- image
- here i changed it level
-  Change 5
- image
-  here i changed it level
@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

## Release Notes

- **New Features**
	- Enhanced layout for various components, improving user interface consistency and responsiveness.
	- Added conditional checks for user roles in patient management, ensuring appropriate access to functionalities.
	- Introduced a confirmation dialog for changing patient code statuses in consent records.

- **Bug Fixes**
	- Improved handling of existing consent records during file uploads to prevent errors.

- **Style**
	- Updated spacing and layout styles across multiple components for better visual presentation.

### CODE REVIEW

File: src/CAREUI/interactive/FiltersSlideover.tsx
Comment (rithviknishad) [inline src/CAREUI/interactive/FiltersSlideover.tsx:1]: let's switch to using `Button` component instead of `ButtonV2`.

File: package-lock.json
Comment (rithviknishad) [inline package-lock.json:1]: discard changes made to the lockfile

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by rithviknishad** [comment]
Closing as duplicate of #9161; Do make changes to the same PR instead of opening new ones. Helps keep track of requested changes better.

File: package-lock.json
Comment (gurramkarthiknetha) [reply package-lock.json:1]: > discard changes made to the lockfile

ok

**Pull request closed without merging**